simulation/sub8_gazebo/worlds/contact.world

```diff
@@ -84,14 +91,18 @@
           </geometry>
         </collision>
       </link>
-    </model>
+    </model> -->
+    <include>
```

Comment:
I had only used the ground plane instead of the transdec to test my mission, did you want to have that be in master?